pom.xml

```diff
@@ -19,6 +19,13 @@
     <maven-javadoc-plugin.version>3.1.0</maven-javadoc-plugin.version>
     <maven-source-plugin.version>3.2.0</maven-source-plugin.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <module.status>integration</module.status>
+    <module.parentVersion>HEAD-jre-SNAPSHOT</module.parentVersion>
+    <variant.jvmVersion>8</variant.jvmVersion>
+    <variant.checkerframework>"checker-qual"</variant.checkerframework>
+    <otherVariant.version>HEAD-android-SNAPSHOT</otherVariant.version>
+    <otherVariant.jvmVersion>6</otherVariant.jvmVersion>
+    <otherVariant.checkerframework>"checker-compat-qual", "version": { "requires": "2.5.5" }</otherVariant.checkerframework>
```

Comment:
Similarly: Do we need to specify the version number here? I see that it's omitted for checker-qual above.

Reply:
The reason why the version is required here, is that the version of `checker-compat-qual` is not managed in the `<dependencyManagement>` block below. It's only defined directly in [android/pom.xml](https://github.com/google/guava/blob/master/android/pom.xml#L255).

I think the better solution us to add it do the <dependencyManagement> block as well. I made that change: 70eaaf7869b89e6c4aa46ce93fa126af80e7334b.
Let me know if that is fine.

Note: The version is then defined in two places, here and in `android/pom.xml`. I am not sure why there are versions in `android/pom.xml` at all because that one also uses this pom as parent. Maybe all the versions can be removed from there?